examples/python/README.md

```diff
@@ -23,16 +21,30 @@ pip install onnx
 pip install onnxruntime
```

Comment:
Since installing `onnxruntime` vs `onnxruntime-gpu` does not impact the model builder, should we tell the user to install `onnxruntime-gpu` instead to make sure they can run their models on GPU as well?